packages/kubernetes-1.20/aws-config

```diff
@@ -0,0 +1,3 @@
+{{~#if settings.kubernetes.credential-providers.aws-config~}}
+{{base64_decode settings.kubernetes.credential-providers.aws-config}}
+{{~/if~}}
```

Comment:
It might be worth supporting both an `aws-config` and an `aws-credentials` file.